src/include/OpenImageIO/typedesc.h

```diff
@@ -295,6 +295,8 @@ struct OIIO_API TypeDesc {
     static const TypeDesc TypeVector;
     static const TypeDesc TypeNormal;
     static const TypeDesc TypeMatrix;
+    static const TypeDesc TypeMatrix33;
+    static const TypeDesc TypeMatrix44;
```

Comment:
Is it worth adding `TypeMatrix44` as an alias of `TypeMatrix`? (There is no `TypePoint3` for example).